projects/rna/in-place-connection/index.jsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useRef } from 'react';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
```

Comment:
By any chance did y'all consider TypeScript for this new project? Or is that a non-starter for y'all?

Reply:
That would be awesome, I love TypeScript.
However, I would rather start with good ol' JS and not complicate it further.
Switching to TS shouldn't be a problem later on, as long as the team agrees.